pennylane/pulse/transmon.py

```diff
@@ -241,8 +241,8 @@ def transmon_drive(amplitude, phase, freq, wires, d=2):
     in :class:`ParametrizedHamiltonian` and have a signature of two parameters, ``(params, t)``.
 
     For realistic simulations, one may restrict the amplitude, phase and drive frequency parameters.
-    For example, the authors in `2008.04302 <https://arxiv.org/abs/2008.04302>`_ impose the restrictions of
-    a maximum amplitude :math:`\Omega_{\text{max}} = 20 \text{MHz}` and the carrier frequency to deviate at most
+    For example, the authors in `2203.06818 <https://arxiv.org/abs/2203.06818>`_ impose the restrictions of
+    a maximum amplitude :math:`\Omega_{\text{max}} = 0.02 \text{GHz}` and the carrier frequency to deviate at most
     :math:`\nu - \omega = \pm 1 \text{GHz}` from the qubit frequency :math:`\omega`
```

Comment:
I'm fairly convinced the authors in [2203.06818](https://arxiv.org/pdf/2203.06818.pdf) didnt mean that these numbers are implicitly multiplied by $2 \pi$ and really are just those numbers in SI

![image](https://user-images.githubusercontent.com/43949391/235139992-81a5552a-c355-4875-8648-019c01ff8979.png)

Reply:
We didn't really resolve this. Can you elaborate on why you believe the amplitude numbers aren't converted from frequency to angular frequency for simulations? Their reference paper is not helpful, it appears to have all amplitudes in arbitrary units. I don't understand why it's cited there.

For now I'm fixing the documentation assuming that anything provided in Hz will be converted to angular frequency when calculating a Hamiltonian. If needed I guess we can have an emergency meeting Monday morning to sort out anything we want to rephrase here.

Reply:
I think they follow the 'ignorant theorist' approach where 0.02Ghz =0.02 10^9/s and nothing is implicitly converted.
The comment is there because I first changed everything to have explicit 2pi factors but I think they didn't intend for that

Reply:
But feel free to ignore this I don't think we have to / can resolve this unambiguously 😄